kubectl-plugin/src/resources/mod.rs

```diff
@@ -1,11 +1,13 @@
 pub mod pool;
+pub mod utils;
 pub mod volume;
 
 use structopt::StructOpt;
 
 pub(crate) type VolumeId = String;
 pub(crate) type ReplicaCount = u8;
 pub(crate) type PoolId = String;
+pub(crate) type OutputFormat = String;
```

Comment:
OutputFormat could be defined in the utils as an enum OutputFormat { Wide,  Yaml, Json }, or something along those lines.

Reply:
OutputFormat has been defined as enum. The StructOpts args is still a string, as we can check the correctness and use it as an enum in the underlying code. Thank you.